receiver/purefareceiver/receiver.go

```diff
@@ -50,8 +50,8 @@ func (r *purefaReceiver) Start(ctx context.Context, compHost component.Host) err
 	scrapeCfgs := []*config.ScrapeConfig{}
 
 	commomLabel := model.LabelSet{
-		"deployment.environment": model.LabelValue(r.cfg.Env),
```

Comment:
I'm curious as to why this is being changed. As it was, this was compliant with the semantic conventions, and this change breaks this.